README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
-- tests are expected to be run in order and some specs are dependent on the state produced by previous specs, for example:
-  - `03-conduct_test_spec.js` uses the patient created in `02-add_patient_spec.js`
-  - `04-get_result_from_patient_link_spec.js` uses the patient link generated in `03-conduct_test_spec.js`
-  - `08-save_and_start_test_spec.js` uses the patient created in `02-add_patient_spec.js` and expects them to not have any tests in progress
-- our tests expect there to only be 1 facility and will fail if there are multiple facilities for whatever org the test user is authenticated into
-  - when running the tests with Okta, the test user Cypress McTestUser has admin access to an org with external ID `DAT_ORG`, which with our default sample data will have 2 facilities
-- `00-health_check.js` is used in our CI to check that the app is up but fails when running locally with either of the commands listed here
-- `01-organization_sign_up_spec.js` is not re-runnable
-- `05-self_registration_spec.js` expects the org your test user is in to have a patient registration link generated
-
-See the [Cypress documentation](https://docs.cypress.io/api/table-of-contents) for writing new tests. If you need to generate new Wiremock mappings for external services, see [this wiki page](https://github.com/CDCgov/prime-simplereport/wiki/WireMock).
-
-
-## Load Tests
-SimpleReport leverages Locust for running its load tests. Currently, Locust starts as part of the local development build, and spawns three separate worker containers for inundating the system.
-
-To compose Locust with the rest of the development stack, run `yarn locust` instead of `yarn start`. *(NOTE: All other prerequisites listed above in "Developing locally" are still required for tests to function.)*
-
-To access the Locust user interface, navigate to `http://localhost:8089/`. Load tests will not kick off automatically; they must be manually invoked.
-
-Tests can be configured by modifying `backend/locust/locustfile.py`.
-
-
-## Twilio
-
-Twilio's Java SDK auto-configures based on two environment variables: `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN`. SMS is also disabled by default, and can be enabled in application.yml:
-
-```
-twilio:
-  enabled: true
-  from-number: +13214560987
-```
-
-These can also be set by environment variable if desired.
-
-## MailHog
-
-MailHog is an email-testing tool with a fake SMTP server underneath, we can use it to test sending emails locally.
-
-- Mailhog client runs on docker, `docker compose up -d mailhog`
-
-- Access mailhog inbox on `http://localhost:8025/`
-
-## Frontend
-
-The front end is a React app. The app uses [Apollo](https://www.apollographql.com/) to manage the graphql API. For styling the app leverages the [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/). Ensure your node version is version 14 in order for
-yarn to build packages correctly.
-
-### Linters
```

Comment:
⚠️ Moved to https://github.com/CDCgov/prime-simplereport/wiki/SimpleReport-Style-Guide as a new section